services.py

```diff
@@ -4,61 +4,75 @@
 
 EON = os.path.isfile('/EON')
 
+RESERVED_PORTS = [8022, 22]  # sshd
+STARTING_PORT = 8001
```

Comment:
test_services.py checked that port > 8000, can we do >= 8000 instead or is that reserved?

Reply:
Not exactly sure. Let's keep 8001 as a starting port.